packages/varnish/src/theme/colors.ts

```diff
@@ -60,6 +60,7 @@ export const color2: Dictionary<Color> = {
   B3: new Color('B3', '#265ED4', true),
   B4: new Color('B4', '#1B4596', true),
   B5: new Color('B5', '#223367', true),
+  B6: new Color('B6', '#265ED461', true), // TODO: validate naming conventions with Jon
```

Comment:
@jonborchardt - what's the naming convention for adding new colors? 
Got this one and N6 below from the Figma here - https://www.figma.com/file/11UobOSIbxrl0zxHgbpjWU/%5E_%5E--*~*~*WIP*~*~*-Varnish-2.0-Component-Library?node-id=10120-139408&t=t5dwM9eFWWBMuDrl-4 

Also how did we decide to name these as "A1", "B1", etc.?

Reply:
<img width="840" alt="Screen Shot 2023-03-16 at 10 43 26 PM" src="https://user-images.githubusercontent.com/6945628/225822329-d7ebd4de-f47e-4492-8ec0-2dd3555e0f6d.png">

this is what the new disabled button colors look like btw ^

Reply:
i have concerns (stated in another comment)
but to answer your q

A1 is just faster than Aqua1
so we have a color.. and 5 shades/tints of it.

Reply:
we decided to revisit this in a separate set of changes so resolving this comment on this pr